esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/install_cold/rose-app.conf

```diff
@@ -3,3 +3,9 @@ default=install_cold.sh
 
 [env]
 SITE=${SITE}
+MODULE_NAME=${MODULE_NAME}
+ESMVALTOOL_DIR = ${CYLC_SUITE_RUN_DIR}/ESMValTool
+ESMVALCORE_DIR = ${CYLC_SUITE_RUN_DIR}/ESMValCore
```

Comment:
These lines will not be needed once the other suggestions are in place:
```suggestion
```

Reply:
[71e9256](https://github.com/ESMValGroup/ESMValTool/pull/2803/commits/71e925628af88ad9120e9942b79316ab7c1a2c56)